PowerShellTools/Intellisense/AutoCompletionController.cs

```diff
@@ -351,5 +357,30 @@ private void DeleteRightBrace()
                 undo.Complete();
             }
         }
+
+        /// <summary>
+        /// Determines whether a command is unhandled.
+        /// </summary>
+        /// <param name="pguidCmdGroup">The GUID of the command group.</param>
+        /// <param name="command">The command.</param>
+        /// <returns>True if it is an unrecognized command.</returns>
+        private static bool IsUnhandledCommand(Guid pguidCmdGroup, VSConstants.VSStd2KCmdID command)
+        {
+            return (pguidCmdGroup != VSConstants.VSStd2K ||
+                    (command != VSConstants.VSStd2KCmdID.TYPECHAR &&
+                     command != VSConstants.VSStd2KCmdID.RETURN &&
+                     command != VSConstants.VSStd2KCmdID.DELETE &&
+                     command != VSConstants.VSStd2KCmdID.BACKSPACE &&
+                     command != VSConstants.VSStd2KCmdID.UNDO &&
+                     command != VSConstants.VSStd2KCmdID.CUT &&
+                     command != VSConstants.VSStd2KCmdID.COMMENT_BLOCK &&
+                     command != VSConstants.VSStd2KCmdID.COMMENTBLOCK &&

```

Comment:
Yes, we check this inside ProcessKeyStroke. I added this purely for easier debugging. Without this, you can barely be able to set a breakpoint as it will run into it millions of times. I use this as a filter.